web/src/compose_state.ts

```diff
@@ -58,6 +59,14 @@ export function has_recipient_viewed_topic_resolved_banner(): boolean {
     return recipient_viewed_topic_resolved_banner;
 }
 
+export function set_recipient_guest_ids_for_dm_warning(guest_names: number[]): void {
+    recipient_guest_ids_for_dm_warning = guest_names;
```

Comment:
`guest_names` is a bad name for an integer; this should be something like `guest_user_ids`.